media-gfx/viewnior/viewnior-1.7-r2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson xdg-utils
+
+MY_PN=Viewnior
+DESCRIPTION="Fast and simple image viewer"
+HOMEPAGE="https://siyanpanayotov.com/project/viewnior/ https://github.com/hellosiyan/Viewnior"
+SRC_URI="https://github.com/hellosiyan/${MY_PN}/archive/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-libs/glib:2
+	media-gfx/exiv2:0=
+	x11-libs/gtk+:2
+"
+DEPEND="${RDEPEND}
+	dev-util/glib-utils
+	virtual/pkgconfig
+	x11-misc/shared-mime-info
+"
+
+S="${WORKDIR}/${MY_PN}-${P}"
+
+PATCHES=(
+	"${FILESDIR}/0.17-Replace-calls-to-getenv-with-g_getenv.patch"
+	"${FILESDIR}/0.17-update_metadata_location.patch"
```

Comment:
Please add the package name to patch file also. I know the getenv one was already in place, but for the new one you're adding here.

Reply:
Fixed.